group_vars/all/mails.yml

```diff
@@ -0,0 +1,27 @@
+---
+##
+# Mailjet smtp configuration (authentication in mails-secrets.yml)
+##
+mailjet__smtp_host: in-v3.mailjet.com
+mailjet__stmp_port: 587
+mailjet__smtp_tls: true
+
+##
+# nullmailer default configuration (using mailjet)
+##
+# Send all mails though mailjet only
+nullmailer__remotes:
+  - >
+    {{ mailjet__smtp_host }}
+    smtp --port={{ mailjet__stmp_port }}
+    {% if mailjet__smtp_tls %}--starttls{% endif %}
+    --user={{ mailjet__api_key }}
+    --pass={{ mailjet__secret_key }}
+
+# All mails to <user>@localhost or <user>@<nullmailer__mailname>
+# are redirected to this email (ie. no local mailbox)
+nullmailer__admin: infra@openfoodfacts.org
+
+# Configure to nullmailer to use host_domain as default mailing domain
+# so unconfigured mail are emited in @<host_domain>
+nullmailer__domain: '{{ host_domain }}'
```

Comment:
why not openfoodfacts.org ?

If we use other domains we will have to configure DKIM and all the things for those subdomains…

Reply:
For most of our servers, {{ host_domain }} will indeed be openfoodfacts.org (or infra.openfoodfacts.org if we stick with this naming scheme). We'll need to configure SPF and DKIM for all the domains we plan to use, so it might be better to standardize and use a few consistent ones. However, I wasn't entirely sure if we should limit ourselves to just one domain (openfoodfacts.org).

Couldn't we consider the possibility of using other domains, such as openfoodfacts.net for some pre-production servers for instance?

Reply:
Normally servers are on openfoodfacts.org, but yes we can discuss it in the future.